mirrors.d/mirror.cenuta.com.yml

```diff
@@ -1,10 +1,15 @@
 ---
-name: mirror-denizli.cenuta.com
+name: mirror.cenuta.com
 address:
-  http: http://mirror-denizli.cenuta.com/almalinux/
-  https: https://mirror-denizli.cenuta.com/almalinux/
+  http: http://mirror.cenuta.com/almalinux
+  https: https://mirror.centos.com/almalinux
```

Comment:
After looking over this again, it appears the https mirror is invalid. Please update this before we merge it in.